R/ingest-scp.R

```diff
@@ -139,45 +122,28 @@ get_incidence_data <- function(state, chosen_cancer, chosen_race, chosen_sex, ch
 #'    "wa",
 #'    current_row
 #' )
-#'
 
 process_row <- function(state, row) {
-  current_cancer <- as.character(row["cancer_type"])
-  current_race <- as.character(row["race"])
-  current_sex <- as.character(row["sex"])
-  current_age <- as.character(row["age"])
-  current_stage <- as.character(row["stage"])
-  current_timespan <- as.character(row["timespan"])
+  row <- map_chr(row, as.character)
 
```

Comment:
Time to learn about purrr! Purrr is the tidyverse's answer to functional programming. To understand how this function works, try:

```r
purrr::map_chr(1:5, as.character)
```